### ISSUE TITLE
Generation failed

### ISSUE THREAD

**Issue by pmcesky** [napari/cookiecutter-napari-plugin#58]
Failed to generate repository with this cookiecutter template. Tried with another python virtual environment and installed all the packages, still unsuccessful. 

![image](https://user-images.githubusercontent.com/18007222/142655606-2ceb19b8-6869-437e-aa6c-9899b84b4fd2.png)

**Comment by tlambert03** [comment]
can you tell us what operating system you're using, and the full path of the directory in which you're creating the cookiecutter?

**Comment by pmcesky** [comment]
> can you tell us what operating system you're using, and the full path of the directory in which you're creating the cookiecutter?

I'm on windows 10, and the full path of the directory is 
C:\Users\pmces\Documents>
And the template is downloaded at 
C:\Users\pmces\.cookiecutters\cookiecutter-napari-plugin

I tried on other diectories and disks on my computer, failed as well, for this same issue.

**Comment by tlambert03** [comment]
thanks!  will try to reproduce and figure it out

**Comment by pmcesky** [comment]
Thanks very much!!!

**Comment by tlambert03** [comment]
can you try one more thing for me, and type `where git` on your command line (where you were running the cookiecutter command)

**Comment by tlambert03** [comment]
one more thing... I haven't been able to reproduce it yet, including on windows... Can you paste the entire process here?  (including all of the inputs you used)?  Does it happen even if you just accept all of the defaults?

**Comment by pmcesky** [comment]
Yeah it failed even with all of the defaults:
![image](https://user-images.githubusercontent.com/18007222/142660052-dedd7e7a-0f3b-494a-8b71-f33b8fbd263c.png)

And "where git" shows nothing on my powershell.

**Comment by pmcesky** [comment]
I tried this on one of my college's computer and it works. 
It seems to be an issue with my computer. 
I merged my D volume last month to C volume, and reinstalled anaconda to C volume, where it was previously on D. I guess this may cause some issues when cookiecutter trying to locate the files. 
Is it possible to set the right path/file location for cookiecutter manually?

**Comment by tlambert03** [comment]
> It seems to be an issue with my computer.

yeah, I tried to reproduce this a number of ways and couldn't.  I will say that you'll probably need to have `git` installed for cookiecutter to work, but in my case, when I uninstalled it, I got a very clear error that no version control system was found.  (not the label syntax incorrect error).  So I'm still unable to tell you what went wrong in your specific case
